src/robusta/core/sinks/pagerduty/pagerduty_sink.py

```diff
@@ -96,17 +196,24 @@ def write_finding(self, finding: Finding, platform_enabled: bool):
             "routing_key": self.api_key,
             "event_action": PagerdutySink.__to_pagerduty_status_type(finding.title),
             "dedup_key": finding.fingerprint,
+            "links": links,
         }
 
         headers = {"Content-Type": "application/json"}
-        response = requests.post(self.url, json=body, headers=headers)
+        response = requests.post(self.events_url, json=body, headers=headers)
         if not response.ok:
             logging.error(
                 f"Error sending message to PagerDuty: {response.status_code}, {response.reason}, {response.text}"
             )
 
+    def write_finding(self, finding: Finding, platform_enabled: bool):
+        if finding.aggregation_key == "ConfigurationChange/KubernetesResource/Change":
```

Comment:
I would put this in a constant, used both here and in the playbook.
So, if someone changes it one day, it doesn't break

Reply:
done